spark_develop.sh

```diff
@@ -12,12 +12,12 @@ function invoke_docker_compose_spark {
 }
 
 function format_namenode_and_datanode {
-    docker-compose -f docker/docker-compose.spark.yml \
-        -p listenbrainzspark run --rm hadoop-master hdfs namenode \
-        -format -nonInteractive -force
+    invoke_docker_compose_spark run --rm hadoop-master \
+        hdfs namenode -format -nonInteractive -force
 }
 
 if [ "$1" == "format" ]; then shift
+	echo "Formatting NameNode..."
```

Comment:
Indent problem here, but looks good to me otherwise.

Reply:
I don't know why it was not showing up in VS Code. But I force indented it.